hasher-matcher-actioner/Makefile

```diff
@@ -11,17 +11,28 @@ REPOSITORY_NAME = hma-lambda-dev
 DOCKER_TAG ?= ${USER} # todo - change to ${USER}-latest?
 DOCKER_URI = $(call shell-or-die,docker images --filter=reference='*/${REPOSITORY_NAME}:${DOCKER_TAG}' --format='{{.Repository}}')
 
+LOCAL_TE_DIR = local_threatexchange # Should match LOCAL_THREAT_EXCHANGE ARG in Dockerfile
 
 all:
 	@echo >&2 "Must specify target. ${FAIL}"
 
 docker:
+	if [ ! -d ${LOCAL_TE_DIR} ] ; then mkdir ${LOCAL_TE_DIR} ; fi
 	docker build -f Dockerfile -t ${REPOSITORY_NAME} .
 
+docker_with_local_te:
+	cp -r ../python-threatexchange ${LOCAL_TE_DIR}
```

Comment:
Would a symlink work better, perhaps? That way, we don't need to keep syncing things between the two copies of threatexchange.

This might make things weird for @Dcallies though. @Dcallies does your setup do symlinks?